tools/r1cs-opt/r1cs-opt.cpp

```diff
@@ -0,0 +1,16 @@
+// tools/r1cs-opt/r1cs-opt.cpp
+
+#include "mlir/IR/DialectRegistry.h"
+#include "mlir/IR/MLIRContext.h"
+#include "mlir/InitAllDialects.h"
+#include "mlir/InitAllPasses.h"
+#include "mlir/Tools/mlir-opt/MlirOptMain.h"
```

Comment:
use `<>` for the mlir includes and move after the project-specific ones

Reply:
For some reason the formatter moves them to the bottom. I think it is sorting alphabetically?